flowzone.yml

```diff
@@ -92,7 +103,6 @@
   - &gitHubCliEnvironment
     # environment variables used by gh CLI
     # https://cli.github.com/manual/gh_help_environment
-    GH_TOKEN: "${{ secrets.FLOWZONE_TOKEN }}"
```

Comment:
I think it would make more sense to provide the token here
```
GH_TOKEN: ${{ steps.gh_token.outputs.token || secrets.FLOWZONE_TOKEN }}
```
and just ensure we only call this anchor from the step context, and not the job. I think the only place we set it at the job context is protect branch anyway.

That avoids having to have two aliases whenever we need to set gh env.

Reply:
See, https://github.com/product-os/flowzone/blob/master/flowzone.yml#L2531-L2533

Reply:
Yeah, just move that to the steps instead of the job, so you can use a single alias.